sklearn/feature_selection/_univariate_selection.py

```diff
@@ -141,7 +141,7 @@ def f_classif(X, y):
     pval : array, shape = [n_features,]
         The set of p-values.
 
-    See Also
+    See also
```

Comment:
Why change this? This is the convention in scikit-learn:

```
(dev) ogrisel@mba scikit-learn % git grep "See Also" | wc -l
     265
(dev) ogrisel@mba scikit-learn % git grep "See also" | wc -l
      33
```

https://scikit-learn.org/stable/developers/contributing.html#guidelines-for-writing-documentation

Reply:
Ah yes, I hesitated when merging `main` in.
I'll revert that.